### ISSUE TITLE
triggers can not be imported using mysql cli

### ISSUE THREAD

**Issue by AubreyHewes** [doutorfinancas/go-mad#45]
Hi, when triggers are dumped (https://github.com/doutorfinancas/go-mad/pull/40) .. they can not be imported using the mysql cli as they contain the current mysql delimiter `;`. 

For example a dumped trigger looks like this:

````sql
--
-- Trigger `upd_check`
--

CREATE TRIGGER upd_check BEFORE UPDATE ON account
       FOR EACH ROW
       BEGIN
           IF NEW.amount < 0 THEN
               SET NEW.amount = 0;
           ELSEIF NEW.amount > 100 THEN
               SET NEW.amount = 100;
           END IF;
       END;

````

Which fails on the `;` within the trigger -- within the IF statement.

According to the mysql documentation the delimiter needs to be changed to allow for this.

> if you use the [mysql](https://dev.mysql.com/doc/refman/5.7/en/mysql.html) program to define a trigger that executes multiple statements, it is necessary to redefine the [mysql](https://dev.mysql.com/doc/refman/5.7/en/mysql.html) statement delimiter so that you can use the ; statement delimiter within the trigger definition. -- https://dev.mysql.com/doc/refman/5.7/en/trigger-syntax.html

So a trigger dump should maybe look like this:

````sql
--
-- Trigger `upd_check`
--
delimiter //
CREATE TRIGGER upd_check BEFORE UPDATE ON account
       FOR EACH ROW
       BEGIN
           IF NEW.amount < 0 THEN
               SET NEW.amount = 0;
           ELSEIF NEW.amount > 100 THEN
               SET NEW.amount = 100;
           END IF;
       END;//
delimiter ;
````

Then the dump can get imported.

With more than one trigger can wrap all triggers with:

````sql
delimiter //
-- triggers; each ending in //
delimiter ;
````

We use the [shopware-cli](https://github.com/FriendsOfShopware/shopware-cli) tool (which uses this library) for creating a dev seed shared between devs and are having to rewrite the seed to accommodate for this.

--
MySQL Version: Server version: 5.7.40 MySQL Community Server (GPL)

https://github.com/FriendsOfShopware/shopware-cli/issues/141

**Comment by AubreyHewes** [comment]
@shyim Is this an issue you could fast track? else using the shopware-cli dump command is useless?

**Comment by shyim** [comment]
I am taking a look :D

**Comment by bolovsky** [comment]
@shyim will close the 2nd PR, since this is done. Will do a release tomorrow with this, after updating the documentation for the new flag

@AubreyHewes from the lib and it's CLI perspective of this project is fixed. I would say @shyim will probably pick this up in a near future from shopware side

**Comment by bolovsky** [comment]
@shyim released 0.3.2 with the change. Hope it helps :)

https://github.com/doutorfinancas/go-mad/releases/tag/0.3.2

**Comment by shyim** [comment]
I will add it later :)
